src/libcec/adapter/RPi/RPiCECAdapterDetection.cpp

```diff
@@ -54,6 +55,10 @@ bool CRPiCECAdapterDetection::FindAdapter(void)
   if ((iResult = vchi_connect(NULL, 0, vchiq_instance)) != VCHIQ_SUCCESS)
     return false;
 
+  bcm_host_init();
+  iResult = vc_cec_set_passive(true);
```

Comment:
would be nicer to rewrite these 4 lines as `return (vc_cec_set_passive(true) == VCHIQ_SUCCESS);`